client/src/Pages/Profile/SideBar/index.js

```diff
@@ -20,17 +21,22 @@ function a11yProps(index) {
 function SideBar() {
   const classes = useStyles();
   const [value, setValue] = useState(0);
+  const [usename, setUsename] = useState('');
 
   const handleChange = (event, newValue) => {
     setValue(newValue);
   };
 
+  function getUserName(username) {
+    setUsename(username);
+  }
+
```

Comment:
why you make this function?  you can directly pass the setUsername its a function